jdbc/src/test/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaDataTest.java

```diff
@@ -353,21 +439,29 @@ private void testGetOfColInfo(
    * Validate resultSet MetaData returned from getSchemas.
    *
    * @param resultSet ResultSet need to be validated.
+   * @param expectedNumRows Expected number of rows.
    * @throws SQLException If an error occurs while retrieving the value.
    */
-  private void testGetSchemasResult(ResultSet resultSet) throws SQLException {
-    final String[] string1 = {"", "testDB", null};
+  private void testGetSchemasResult(ResultSet resultSet, int expectedNumRows) throws SQLException {
+    final String[] string1 = {"testDB", null};
+    final String[] string2 = {"exampleDB", null};
     final List<String[]> strings = new ArrayList<>();
     strings.add(string1);
+    strings.add(string2);
 
     int numRows = 0;
     while (resultSet.next()) {
+      int match = 0;
       for (int i = 1; i <= resultSet.getMetaData().getColumnCount(); ++i) {
-        Assertions.assertEquals(strings.get(numRows)[i], resultSet.getString(i));
+        if (strings.get(numRows)[i-1] == resultSet.getString(i)) {
```

Comment:
nit. some comments here explaining the test might be helpful.
nit. I would prefer to rewrite this using streams (a more functional style), which could be easier to read.